api-test.json

```diff
@@ -50108,265 +50117,213 @@
-																"value": "application/json",
+																"value": "application/ld+json",
 																"type": "default"
-															},
-															{
-																"key": "Accept",
-																"value": "application/json",
-																"type": "default",
-																"disabled": true
 															}
 														],
 														"body": {
 															"mode": "raw",
-															"raw": "{\r\n\t\"id\": \"urn:ngsi-ld:ContextSourceRegistration:cccc\",\r\n\t\"type\": \"ContextSourceRegistration\",\r\n\t\"information\": [{\r\n\t\t\t\"entities\": [{\r\n                \"id\": \"urn:test:testentity12345789\",\r\n\t\t\t\t\"type\": \"Test2\"\r\n\t\t\t}],\r\n            \"propertyNames\": [\"brandName\",\"speed\"],\r\n            \"relationshipNames\": [\"isParked\"]\r\n\t\t}\r\n\t],\r\n\t\"endpoint\": \"http://localhost:9080\",\r\n    \"mode\": \"exclusive\",\r\n    \"operations\": [\"retrieveEntity\", \"createEntity\", \"queryEntity\"]\r\n}",
-															"options": {
-																"raw": {
-																	"language": "json"
-																}
-															}
+															"raw": "{\r\n    \"intproperty-new\": {\"type\": \"Property\", \"value\": 123},\r\n     \"@context\": \"https://raw.githubusercontent.com/ScorpioBroker/ScorpioBroker/new_ci/testcontext.json\"\r\n}"
 														},
 														"url": {
-															"raw": "{{registry-manager}}/ngsi-ld/v1/csourceRegistrations",
+															"raw": "{{entity-manager}}/ngsi-ld/v1/entities/urn:notfound/attrs",
 															"host": [
-																"{{registry-manager}}"
+																"{{entity-manager}}"
 															],
 															"path": [
 																"ngsi-ld",
 																"v1",
-																"csourceRegistrations"
+																"entities",
+																"urn:notfound",
+																"attrs"
 															]
 														}
 													},
 													"response": []
 												},
 												{
-													"name": "Create all matching reg simple Entity",
+													"name": "Update attribute value if entity id is not found",
 													"event": [
 														{
 															"listen": "test",
 															"script": {
 																"exec": [
-																	"var payloadBody = JSON.parse(pm.request.body.raw);\r",
-																	"console.log(payloadBody.id);\r",
-																	"var entityId = payloadBody.id;\r",
-																	"pm.collectionVariables.set(\"created\", payloadBody);\r",
-																	"pm.collectionVariables.set(\"entityId\", entityId);\r",
-																	"pm.test(\"Status code is 422\", function () {\r",
-																	"    pm.response.to.have.status(422);\r",
+																	"var expected = {\r",
+																	"    \"type\": \"https://uri.etsi.org/ngsi-ld/errors/ResourceNotFound\",\r",
+																	"    \"title\": \"Resource not found.\",\r",
+																	"    \"errorCode\": 404\r",
+																	"};\r",
```

Comment:
Add test case for partial update if attribute not found.

Reply:
done